include/mega/types.h

```diff
@@ -358,45 +356,18 @@ enum SyncError {
     REMOTE_NODE_INSIDE_RUBBISH = 20,        // Attempted to be added in rubbish
     VBOXSHAREDFOLDER_UNSUPPORTED = 21,      // Found unsupported VBoxSharedFolderFS
     LOCAL_PATH_SYNC_COLLISION = 22,         // Local path includes a synced path or is included within one
-    LOCAL_IS_FAT = 23,                      // Found FAT (not a failure per se)
-    LOCAL_IS_HGFS= 24,                      // Found HGFS (not a failure per se)
     ACCOUNT_BLOCKED= 25,                    // Account blocked
     UNKNOWN_TEMPORARY_ERROR = 26,           // Unknown temporary error
     TOO_MANY_ACTION_PACKETS = 27,           // Too many changes in account, local state discarded
     LOGGED_OUT = 28,                        // Logged out
 };
 
-inline bool isSyncErrorPermanent(SyncError e)
-{
-    switch (e)
-    {
-    case NO_SYNC_ERROR:
-    case LOGGED_OUT: //syncs may be restored after relogging
-    case UNKNOWN_TEMPORARY_ERROR:
-    case STORAGE_OVERQUOTA:
-    case BUSINESS_EXPIRED:
-    case FOREIGN_TARGET_OVERSTORAGE:
-    case ACCOUNT_BLOCKED:
-    case LOCAL_IS_FAT:
-    case LOCAL_IS_HGFS:
-        return false;
-    default:
-        return true;
-    }
-}
+enum SyncWarning {
+    NO_SYNC_WARNING = 0,
+    LOCAL_IS_FAT = 23,                      // Found FAT (not a failure per se)
+    LOCAL_IS_HGFS= 24,                      // Found HGFS (not a failure per se)
```

Comment:
There's a _problem_ with separate warnings from errors in enumeration: if we ever start considering these an error (i.e. we cease to support that), we'd need to adjust apps accordingly. Not a big issue anyway ;-)

Reply:
I did that mainly so we didn't have to call isError() all the time - presumably the apps would need to have a version of isError() too under the old system, and so they would need to be adjusted regardless.